pkgs/servers/http/apache-httpd/2.2.nix

```diff
@@ -23,6 +23,7 @@ stdenv.mkDerivation rec {
   # FIXME: -dev depends on -doc
   outputs = [ "dev" "out" "doc" ];
   setOutputFlags = false; # it would move $out/modules, etc.
+  stripDebugList= [ "bin" "lib" "modules" ]; # also strip modules.
```

Comment:
I could of course unify the other way around, but I feel this makes more sense with the name of the variable, and with the other pull requests I just made.